ayon_server/settings/overrides.py

```diff
@@ -169,42 +169,66 @@ def extract_overrides(
     explicit_pins: list[list[str]] | None = None,
     explicit_unpins: list[list[str]] | None = None,
 ) -> dict[str, Any]:
+    existing_overrides = {**existing} if existing else {}
```

Comment:
Is this just trying to make a copy?
```suggestion
    existing_overrides = existing.copy() if existing else {}
```

Also be aware that this will not be a `deepcopy` as such the input may still get affected.
So maybe this is safer:
```suggestion
    existing_overrides = copy.deepcopy(existing) if existing else {}
```

Reply:
Nevermind - this seems redundant copy because we're not editing the `existing_overrides` down below - we're solely using it as lookup. Which makes me wonder why we need the `{**existing}`?

Reply:
yep. it was originally modified, but it is no longer a case. fixed